src/preprocessing/passes/non_clausal_simp.h

```diff
@@ -48,6 +49,29 @@ class NonClausalSimp : public PreprocessingPass
 
   /** Learned literals */
   std::vector<Node> d_nonClausalLearnedLiterals;
+
+  /** Preprocess the boolean skeleton using CryptoMinisat
+      Args:
+      - assertionsToPreprocess: Top level assertions
+      - substs_index: the substitution index in the current context
+      Return:
+      whether the boolean skeleton is not satisfiable and the
+      learned unit clauses
+   */
+  std::pair<bool, std::vector<Node>> preprocessByCryptoMinisat(
+      AssertionPipeline* assertionsToPreprocess, unsigned substs_index);
+
+  /** Preprocess the boolean skeleton using CryptoMinisat
+      Args:
```

Comment:
Use [doxygen](http://www.stack.nl/~dimitri/doxygen/)-style comments for describing arguments and return values.